src/app/manageConsole/views/howtoEditServiceConfig.ejs

```diff
@@ -0,0 +1,354 @@
+        </p>
+        <p class="govuk-body">
+            Find out more about <a  class="govuk-link" href="https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/">client ID on OAuth</a>.
+        </p>
+        <h2 id="redirect-url-id" class="govuk-heading-s ">Redirect URL</h2>
+       
+        <p class="govuk-body">
+            The redirect URL is where you want to redirect users to after they have authenticated. DfE Sign-in will send the authentication response to this URL. 
+        </p>
+        <p class="govuk-body">
+            You can add multiple redirect URLs. For security reasons, it is important to add at least 1. 
+        </p>
+        <p class="govuk-body ">
+            URLs must be valid, trusted and start with <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https</span>. URLs must not include:
+        </p>
+        <ol class="govuk-list govuk-list--bullet ">
+            <li>query string, for example <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https://myservice.com/auth/cb?user=xyz</span></li>
+            <li>local host, for example <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https://localhost</span></li>
+            <li>wildcards, for example <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https://*.myservice.com</span></li>
+        </ol>
+        <p class="govuk-body">
+            Users will only be redirected back to a URL if it exists in this whitelist.
+        </p>
+        <p class="govuk-body">
+            It is important to keep redirect URLs up to date and remove any URLs that are no longer relevant. You can remove a redirect URL on the ‘Edit service configuration’ page by clicking ‘Remove’ next to the URL field.
+        </p>
+        <h2 id="logout-redirect-url-id" class="govuk-heading-s ">Logout redirect URL</h2>
+        <p class="govuk-body">
+            The logout redirect URL is the page that you want to redirect users to after they log out of the service.
+        </p>
+        <p class="govuk-body">
+            It is usually the service landing page from DfE Sign-in. For example,  
+        </p>
+        <p class="govuk-body">
+           <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https://www.apply-for-teacher-training.service.gov.uk/provider</span>
+        </p>
+        <p class="govuk-body">
+            You can add multiple redirect URLs. For security reasons, it is important to add at least 1. 
+        </p>
+        <p class="govuk-body ">
+            URLs must be valid, trusted and start with <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https</span>. URLs must not include:
+        </p>
+        <ol class="govuk-list govuk-list--bullet ">
+            <li>query string, for example <span style="background-color: rgb(235, 232, 232); color:#d4351c;">https://myservice.com/auth/cb?user=xyz</span></li>
```

Comment:
it should be auth/cb?clientid Reference: (https://github.com/DFE-Digital/login.dfe.oidc/blob/develop/README.md#getting-started)